src/handlers/submitHandler.ts

```diff
@@ -0,0 +1,52 @@
+import { IHttp, IModify, IPersistence, IRead } from '@rocket.chat/apps-engine/definition/accessors';
+import { UIKitViewSubmitInteractionContext , UIKitInteractionContext} from '@rocket.chat/apps-engine/definition/uikit';
+import { ModalsEnum } from '../enums/Modals';
+import { MiroApp } from '../../MiroApp';
+import { SlashCommandContext } from '@rocket.chat/apps-engine/definition/slashcommands';
+import { getUIData } from '../lib/persistence';
+import { getBoards } from '../lib/get/getBoards';
+
+export class ExecuteViewSubmitHandler {
+	constructor(
+		private readonly app: MiroApp,
+		private readonly read: IRead,
+		private readonly http: IHttp,
+		private readonly modify: IModify,
+		private readonly persistence: IPersistence,
+        
+	) {}
+
+	public async run(
+        app:MiroApp,
+        context: UIKitViewSubmitInteractionContext,
+        read: IRead,
+        http: IHttp,
+        persistence: IPersistence,
+        modify: IModify,
+        slashcommandcontext?: SlashCommandContext, 
```

Comment:
Do we need these `slashCommandContext` and `uiKitContext` params here? They're not being used anywhere.